Libraries/LibGUI/GDirectoryModel.cpp

```diff
@@ -119,7 +119,7 @@ GModel::ColumnMetadata GDirectoryModel::column_metadata(int column) const
 {
     switch (column) {
     case Column::Icon:
-        return { 16, TextAlignment::Center };
+        return { 16, TextAlignment::Center, false };
```

Comment:
This API is not very expressive, and I think we can do a little bit better.
Let's make an enum for this, something like
```cpp
enum class AllowSorting {
    No,
    Yes,
};
```

Then the calling code will be much easier to understand:
```cpp
return { 16, TextAlignment::Center, AllowSorting::No };
```